lint/linter.py

```diff
@@ -1127,7 +1127,6 @@ def clear_view(view):
 
         view.erase_status(STATUS_KEY)
         highlight.clear_view(view)
-        persist.errors.pop(view.id(), None)
```

Comment:
`clear_view`, `clear_all` and `clear` should go somewhere, prob to hell?. 

But we can have a different PR for this stuff. Probably an issue so we don't forget.

Reply:
#899